datatable/extras/__init__.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+# © H2O.ai 2018; -*- encoding: utf-8 -*-
+#   This Source Code Form is subject to the terms of the Mozilla Public
+#   License, v. 2.0. If a copy of the MPL was not distributed with this
+#   file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#-------------------------------------------------------------------------------
+
+__all__ = ("aggregate", )
```

Comment:
Generally, in order to export a symbol, you need to import or define it first. 
However in your case (allow Python to find `datatable.extras.aggregate`) having an empty `__init__.py` should work just fine.

Reply:
Yes, I was about to remove this line. Thanks!